src/components/look-controls.js

```diff
@@ -290,16 +292,17 @@ module.exports.Component = registerComponent('look-controls', {
   /**
    * Register mouse down to detect mouse drag.
    */
-  onMouseDown: function (evt) {
+  onMouseDown: function (event) {
     var sceneEl = this.el.sceneEl;
     if (!this.data.enabled || (sceneEl.is('vr-mode') && sceneEl.checkHeadsetConnected())) { return; }
     // Handle only primary button.
-    if (evt.button !== 0) { return; }
+    if (event.button !== 0) { return; }
 
     var canvasEl = sceneEl && sceneEl.canvas;
 
     this.mouseDown = true;
-    this.previousMouseEvent = evt;
```

Comment:
This makes some sense. We're holding a reference to the event object that somehow gets deallocated or values get overwritten by the browser. I still don't understand why it does only happen in some Firefox instances / computers. Wondering if there's a deeper issue that could affect other parts of the code base.

Reply:
A cursory search for `= evt;` reveals the same thing may occur in *src/components/cursor.js* but it's not an issue I have noticed.

Reply:
Correction, in *src/components/cursor.js* the event is not kept beyond the function therefore won't present an issue.